test/coinselector.spec.ts

```diff
@@ -0,0 +1,72 @@
+import { Coin } from '../src/wallet/coin';
+import { CoinSelector } from '../src/wallet/coin-selector.ts';
+import { Transaction } from 'bitcoinjs-lib';
+
+describe('CoinSelector', () => {
+    let coinSelector: CoinSelector;
+    const feeRate: number = 5;
+
+    beforeAll(() => {
+        coinSelector = new CoinSelector(feeRate);
+    });
+
+    it('should generate correct cost of change', () => {
+        const outputSize = 31;
+        const inputSizeOfChangeUTXO = 68;
+
+        // Calculate cost of change based on output size and input size
+        const costOfChangeOutput = outputSize * feeRate;
+        const costOfSpendingChange =
+            inputSizeOfChangeUTXO * CoinSelector.LONG_TERM_FEERATE;
+        const correctCostOfChange = costOfChangeOutput + costOfSpendingChange;
```

Comment:
We dont need to do the calculation in the test. we can just have `expectedCostOfChange = value`